lib/app.js

```diff
@@ -78,28 +81,8 @@ export class Container extends Component {
   }
 
   render () {
-    const { hasError } = this.context._containerProps
-
-    if (hasError) {
-      return null
-    }
-
     const {children} = this.props
-
-    if (process.env.NODE_ENV === 'production') {
-      return <>{children}</>
-    } else {
-      const ErrorDebug = require('./error-debug').default
-      const { AppContainer } = require('react-hot-loader')
-
-      // includes AppContainer which bypasses shouldComponentUpdate method
-      // https://github.com/gaearon/react-hot-loader/issues/442
-      return (
-        <AppContainer warnings={false} errorReporter={ErrorDebug}>
-          {children}
-        </AppContainer>
-      )
-    }
+    return <>{children}</>
```

Comment:
I think you can remove the empty fragment here and just `return children`? or is there a clever reason as to why wrap it in a fragment?

Reply:
Just didn't want to change behaviour from what it was before.

Reply:
Sounds good 👍